coq-builtin.elpi

```diff
@@ -934,6 +934,13 @@ external pred coq.hints.set-opaque i:constant, i:string, i:bool.
 % transparent (ff) in DB
 external pred coq.hints.opaque i:constant, i:string, o:bool.
 
+% [coq.hints.add-resolve GR DB Priority Pattern] Like Hint Resolve GR |
+% Priority : DB.
+% Supported attributes:
+% - @local! (default is export)
+% - @global! (discouraged, may become deprecated)
```

Comment:
@ejgallego I don't see an `@export!` macro in [coq-builtin.elpi](https://github.com/LPCIC/coq-elpi/blob/master/coq-builtin.elpi); do you plan to add this, and how should the `#[export]` attributed be added to without such a macro?

Reply:
It is the default (if you say nothing, that is what you get). BTW, I'm @gares ;-)

Reply:
Haha, of course :upside_down_face:

I had a total brain fart; apologies for the noise